devtools/src/main/scala/kalix/devtools/impl/KalixProxyContainer.scala

```diff
@@ -80,10 +76,10 @@ class KalixProxyContainer private (image: DockerImageName, config: KalixProxyCon
     Seq("-Dconfig.resource=dev-mode.conf", "-Dlogback.configurationFile=logback-dev-mode.xml")
 
   val pubSubContainerArg =
-    if (config.pubsubEmulatorHost.nonEmpty) {
-      withEnv("PUBSUB_EMULATOR_HOST", config.pubsubEmulatorHost)
-      Some("-Dkalix.proxy.eventing.support=google-pubsub-emulator")
-    } else None
+    config.pubsubEmulatorPort.map { port =>
+      withEnv("PUBSUB_EMULATOR_HOST", "host.testcontainers.internal")
+      "-Dkalix.proxy.eventing.support=google-pubsub-emulator"
+    }
```

Comment:
When running it from kalix:run, we should do it differently. The pubsube will be running through docker-compose. And the proxy container will run inside testcontainrs. 

Therefore, we must configure it to reach pubsub on the container host + port.